internal/i18n/errors.go

```diff
@@ -29,5 +29,8 @@ func NewError(ctx context.Context, msg MessageKey, inserts ...interface{}) error
 
 // WrapError wraps an error
 func WrapError(ctx context.Context, err error, msg MessageKey, inserts ...interface{}) error {
+	if err == nil {
```

Comment:
This kept catching me out in the code - where we'd return a `nil` error, if we accidentally used `WrapError` instead of `NewError`. So I've made it so you can safely use `WrapError` for a case where `err` is nil - and it will just fall back to `NewError`.

Reply:
I'm on the fence with this one... it changes the function from "if there was an error, wrap it and return it" to "there was definitely an error, so either wrap this one or create a new one". But I suppose either is fine, as long as all usages expect the latter/new behavior...

Reply:
Assuming we keep this new behavior, we might consider renaming it to `NewWrappedError` or similar. I think `WrapError` is potentially confusing since it now intentionally behaves differently from `errors.Wrap`.